runtime/include/localeModels/knl

```diff
@@ -0,0 +1 @@
+numa
```

Comment:
Oh, github shows this in a strange way.  It's actually just a symbolic link from knl -> numa.  I didn't need to change anything from numa for this directory, but our build system picks which directory to use based on the locale model name, so I made a symbolic link.